src/main/java/net/quarrymod/blockentity/machine/tier3/QuarryBlockEntity.java

```diff
@@ -165,7 +182,7 @@ public void resetOnPlaced() {
     public void tick(World world, BlockPos pos, BlockState state, MachineBaseBlockEntity blockEntity2) {
```

Comment:
Dit ding doet altijd zijn code uitvoeren. 
Idee om dit te fixen dat alleen code uitvoerd als het tijd is om een block te minen.